realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/ProgressListenerTests.kt

```diff
@@ -59,44 +60,39 @@ class ProgressListenerTests {
 
     @After
     fun tearDown() {
-        if (this::realm.isInitialized) {
-            realm.close()
-        }
         if (this::app.isInitialized) {
             app.close()
         }
         RealmLog.setLevel(LogLevel.WARN)
     }
 
-    @Ignore("See https://mongodb.slack.com/archives/CQLDYRJ3V/p1587563930459100")
     @Test
     fun downloadProgressListener_changesOnly() {
         val allChangesDownloaded = CountDownLatch(1)
-        val user1: User = app.login(Credentials.anonymous())
+        val user1: User = app.registerUserAndLogin(TestHelper.getRandomEmail(), "123456")
         val user1Config = createSyncConfig(user1)
-        createRemoteData(user1Config)
-        val user2: User = app.login(Credentials.anonymous())
+        val user2: User = app.registerUserAndLogin(TestHelper.getRandomEmail(), "123456")
         val user2Config = createSyncConfig(user2)
-        val realm = Realm.getInstance(user2Config)
-        val session: SyncSession = realm.syncSession
-        session.addDownloadProgressListener(ProgressMode.CURRENT_CHANGES) { progress ->
-            RealmLog.error(progress.toString())
-            if (progress.isTransferComplete) {
-                assertTransferComplete(progress, true)
-                assertEquals(TEST_SIZE, getStoreTestDataSize(user2Config))
-                allChangesDownloaded.countDown()
+        Realm.getInstance(user2Config).use { realm ->
+            createRemoteData(user1Config)
+            realm.syncSession.addDownloadProgressListener(ProgressMode.CURRENT_CHANGES) { progress ->
+                RealmLog.error(progress.toString())
```

Comment:
Looks like a mistake in the old test. This isn't used for anything.

Reply:
I can remove it. Is it a general pattern to avoid log statements in tests. There are other more complicated ones that seems more essential for analyzing test failures. Maybe the should just be using Android's log instead with a different tag to not cause confusion? Or should it all go

Reply:
We probably left them here because of issues with them and instead of trying to reproduce them with logs enabled it was easier to try to infer the behavior from the logs the first time around.

Generally, we shouldn't do it, but I wouldn't be strongly opposed to keeping them here.

Reply:
I removed the trivial ones, but left a single one here and there.